IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml

```diff
@@ -112,14 +111,15 @@ paths:
         - in: query
           name: embed
           description: >-
-            Influences whether referenced resources are returned in full or as references only
+            Influences whether referenced resources are returned in full or as references only. If not specified, none of the nested resources will be embedded
           schema:
             type: array
             items:
               type: string
               enum:
                 - ALL
-                - LOCATIONS
+                - NONE
+              default: NONE
```

Comment:
as the mandatory implementation is ALL the default should also be ALL

Reply:
Correct. Fixed in all occurrences of embed in next pull request